build.gradle.kts

```diff
@@ -46,22 +46,24 @@ object Versions {
 }
 
 dependencies {
+    // https://mvnrepository.com/artifact/io.provenance/proto-kotlin
+    implementation("io.provenance:proto-kotlin:${Versions.ProvenanceProtos}")
 
     // Kotlin
     implementation("org.jetbrains.kotlin:kotlin-allopen:${Versions.Kotlin}")
     implementation("org.jetbrains.kotlin:kotlin-reflect:${Versions.Kotlin}")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Versions.Kotlin}")
 
     // Provenance
-    implementation("io.provenance.protobuf:pb-proto-java:${Versions.ProvenanceProtos}")
+//    implementation("io.provenance.protobuf:pb-proto-java:${Versions.ProvenanceProtos}")
     implementation("io.provenance.hdwallet:hdwallet:${Versions.ProvenanceHDWallet}")
 
     // Grpc
     implementation("io.grpc:grpc-alts:${Versions.Grpc}")
     implementation("io.grpc:grpc-netty:${Versions.Grpc}")
     implementation("io.grpc:grpc-protobuf:${Versions.Grpc}")
     implementation("io.grpc:grpc-stub:${Versions.Grpc}")
-
+    implementation("com.google.protobuf:protobuf-java:3.19.1")
```

Comment:
Can `3.19.1` be moved into a `Versions.Protobuf`?

Reply:
that dep is no longer required..removed